pytest_splunk_addon_ui_smartx/components/table.py

```diff
@@ -487,15 +487,19 @@ def get_count_number(self):
         row_count = self.get_count_title()
         return int(re.search(r"\d+", row_count).group())
 
-    def get_more_info(self, name, cancel=True):
+    def get_more_info(self, name, cancel=True, expand_row=True):
```

Comment:
I don't get the point of this flag. It seems like leaking abstraction for the method that should return the content of _More Info`. Why should caller of this method know about if it is open or close?

Here is my suggestion
1. check if the row is expanded with attribute `aria-expanded`.
2. if the value is `'false'`, click on the arrow
3. if there is a `Loading` inside, wait it disappear (in case the custom row is used), it loads in runtime 


Here is how I would write in UI unit tests
```ts
    const arrow = within(inputRow).getByRole('cell', { name: /expandable/i });
    const isExpanded = arrow.getAttribute('aria-expanded');
    if (isExpanded === 'false') {
        await userEvent.click(arrow);
    }
    await waitFor(() => expect(arrow.getAttribute('aria-expanded')).not.toBe('false'));
    const loading = screen.queryByText('Loading...');
    if (loading) {
        await waitForElementToBeRemoved(loading);
    }
```

Reply:
We added the flag to ensure that the `moreInfo` section remains expanded based on the use case. This allows any changes made to the input's row to be immediately reflected in the custom row section without collapsing and re-expanding the `moreInfo` section.

Reply:
I have written the unit test case in my PR itself, so we don't need this change in smartx. Hence, we can close this PR.